packages/palette/src/utils/usePosition.ts

```diff
@@ -44,7 +45,7 @@ export const usePosition = ({
   const tooltipRef = useRef<HTMLElement | null>(null)
   const anchorRef = useRef<HTMLElement | null>(null)
 
-  useLayoutEffect(() => {
```

Comment:
Or maybe just use `useEffect`? Is there a noticeable difference between the two UX-wise?